### PR TITLE
Added CDDLv1.1 

### PR DESCRIPTION
**Pull request by sunilk747**

### CODE REVIEW

File: tests/synonyms/license_synonyms.json [hunk 1]
```diff
@@ -197,7 +197,7 @@
     "GNU AGPLv3+ or BSD": "unknown", 
     "for any purpose": "public domain", 
     "Mozilla Public License, Version 2.0": "unknown", 
-    "Common Development and Distribution License (CDDL) v1.0": "unknown", 
+    "Common Development and Distribution License (CDDL) v1.0": "cddlv1.1+",
```
Comment (sara-02) [inline tests/synonyms/license_synonyms.json]: We would need a similar statement for v2 as well.

File: tests/synonyms/license_synonyms.json [hunk 2]
```diff
@@ -325,7 +325,7 @@
     "3-clause BSD": "bsd-new", 
     "Apache Software License 2.0": "apache 2.0", 
     "see LICENSE file": "unknown", 
-    "COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0": "unknown", 
+    "COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0": "cddlv1.1+",
```
Comment (sara-02) [inline tests/synonyms/license_synonyms.json]: Same here

File: tests/test_license_analysis.py [hunk 3]
```diff
@@ -42,6 +42,12 @@ def test_compute_rep_license_successful():
     assert output['status'] == 'Successful'
     assert output['representative_license'] == 'bsd-new'
 
+    list_licenses = ['MIT', 'BSD', 'PD', 'APACHE', 'CDDL 1.0']
+    output = license_analyzer.compute_representative_license(list_licenses)
+    assert output['status'] == 'Successful'
+    assert output['representative_license'] == 'cddlv1.1+'
+    assert set(output['outlier_licenses']) == set(['cddlv1.1+'])
```
Comment (sara-02) [inline tests/test_license_analysis.py]: How are we having `cddlv1.1+` as both a representative and an outlier. Looks like an change in the outlier logic is needed.

**Comment by sara-02** [comment]
@sunilk747 a test for conflict case would be good to add.

**Comment by pkajaba** [comment]
@sunilk747 would you kindly rebase?

File: tests/synonyms/license_synonyms.json (hunk 1 above)
Comment (sunilk747) [reply tests/synonyms/license_synonyms.json]: added for v2.

File: tests/test_license_analysis.py (hunk 3 above)
Comment (sunilk747) [reply tests/test_license_analysis.py]: I think we are good on this. No change is needed as per the discussion.

File: tests/synonyms/license_synonyms.json (hunk 2 above)
Comment (sunilk747) [reply tests/synonyms/license_synonyms.json]: done

**Comment by sunilk747** [comment]
@sara-02 can you merge this when the test passes. Test it once locally on your machine before merging.

**Comment by sara-02** [comment]
@sunilk747 Thanks.

**Pull request merged**